Added Bengali Translation

rabbihossain opened the pull request:
Tried to add Bengali translation.

rabbihossain committed:
Added Bengali Translation

Tried to add Bengali translation.

tamcap commented on res/values-bn-rBD/strings.xml:
```diff
@@ -0,0 +1,56 @@
+    <string name="no">না</string>
+
+    <string name="about_version">সংস্করন %1$s</string>
+    <string name="about_mapbox">ম্যাপিং প্রোভাইডার সম্পর্কে আরও জানুনż/string>
```
You need to replace all instances of ż with < or it won't compile correctly. (1/2)

tamcap commented on res/values-bn-rBD/strings.xml:
```diff
@@ -0,0 +1,56 @@
+
+    <string name="about_version">সংস্করন %1$s</string>
+    <string name="about_mapbox">ম্যাপিং প্রোভাইডার সম্পর্কে আরও জানুনż/string>
+    <string name="about_more">আরওż/string>
```
And here. (2/2)

cpeterso commented:
@rabbihossain: Here are the error messages (from the Travis-CI build) that @tamcap is referring to:

```
type "string" must be terminated by the matching end-tag "</string>".
MozStumbler/res/values-bn-rBD/strings.xml:55:2: Error: The element type "string" must be terminated by the matching end-tag "</string>".
```

cpeterso commented:
Also, when you've fixed and committed your value.xml file, can you please use `git rebase` to squash your branch's commits into a single commit? Here are instructions for squashing git commits: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

rabbihossain committed:
Update strings.xml for bengali localization

rabbihossain committed:
fixed spelling mistake and added more translation

rabbihossain commented:
@cpeterso 
I have fixed the issues and added some more translation in strings.xml . But I didn't understand how to do `git rebase` . As I am noob and I don't have git client installed on my pc.

cpeterso commented:
Thanks for your help, @rabbihossain!
